src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -122,6 +122,11 @@ function createPaymentRequest(args) {
       }
     }
 
+    //Set Apple Pay tokenisation
+    if (AdyenConfigs.getAdyenApplePayTokenisationEnabled() && AdyenHelper.isApplePay(paymentRequest.paymentMethod.type)) {
```

Comment:
Maybe just compare against the applePay constant rather than using a helper function for a string comparison?

Reply:
Yes, since I had to compare it multiple times, I created a helper function for it. Will add the constant in the helper function though